app/src/main/java/org/matrix/matrixandroidsdk/activity/RoomActivity.java

```diff
@@ -61,7 +61,8 @@
  */
 public class RoomActivity extends MXCActionBarActivity implements MatrixMessageListFragment.MatrixMessageListFragmentListener {
 
-    public static final String EXTRA_ROOM_ID = "org.matrix.matrixandroidsdk.RoomActivity.EXTRA_ROOM_ID";
+    public static final String EXTRA_ROOM_ID         = "org.matrix.matrixandroidsdk.RoomActivity.EXTRA_ROOM_ID";
```

Comment:
Please keep to the style of other constants in this file. Do not align the value with other constant values.

The main reason for discouraging this is to avoid the maintenance burden of having to align all your constant values (which may be trivial initially, but quickly becomes annoying when you want to change constant names, etc).